src/marketing/price-rule-multiple-sku.md

```diff
@@ -2,53 +2,56 @@
 title: Price Rule with Multiple SKUs
 ---
 
-A single price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions to match any products of selected SKUs. When building the rule, you can easily browse and select SKUs without needing to know all of your product SKUs from memory.
+A single catalog price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions that match the selected SKUs. When building the rule, you can easily browse and select SKUs from the grid.
 
 ## Step 1. Verify Storefront Properties of Attribute
 
-Before you begin, make sure that the Storefront Properties of the SKU attribute are set to `Use in Promo Rules`.
+Before you begin, make sure that the Storefront Properties of the `sku` attribute are set to `Use in Promo Rules`.
 
 1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
 
-1. In the search filter at the top of the **Attribute** Column, enter `sku` and click <span class="btn">Search</span>.
+1. In the search filter at the top of the **Attribute Code** column, enter `sku` and click <span class="btn">Search</span>.
 
-1. Click to open the attribute in edit mode.
+1. Click to open the `sku` attribute in edit mode.
 
 1. In the left panel, choose **Storefront Properties**. Then, make sure that **Use for Promo Rule Conditions** is set to `Yes`.
 
 1. If you changed the value of the property, click <span class="btn">Save Attribute</span>.
 
 ## Step 2. Apply a Price Rule to Multiple SKUs
 
-1. On the _Admin_ sidebar, click **Marketing**. > _Promotions_ > **Cart Price Rules**.
+1. On the _Admin_ sidebar, click **Marketing**. > _Promotions_ > **Catalog Price Rules**.
 
 1. Do one of the following:
 
-    - Follow the instructions to create a new [cart price rule]({% link marketing/price-rules-cart.md %}).
-    - Open an existing cart price rule.
+    - Follow the instructions to create a new [catalog price rule]({% link marketing/price-rules-catalog.md %}).
+    - Open an existing catalog price rule.
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Conditions** section, and do the following:
 
     - In the first line, set the first parameter to `ANY`.
 
-        ![Cart price rule condition - ANY]({% link images/images/price-rule-catalog-multiple-skus-condition1.png %}){: .zoom}
+        ![Catalog price rule condition - ANY]({% link images/images/price-rule-catalog-multiple-skus-condition1.png %}){: .zoom}
         _If ANY of these conditions are TRUE_
 
     - Click **Add** (![Add icon]({% link images/images/btn-add-grn.png %})) at the beginning of the next line. Then, in the list under **Product Attribute**, choose `SKU`.
```

Comment:
```suggestion
    - Click the **Add** (![Add icon]({% link images/images/btn-add-grn.png %})) icon at the beginning of the next line. Then, in the list under **Product Attribute**, choose `SKU`.
```

Reply:
This one is OK as is. If you add "the", then you also have to add the word "icon" after the link text.  The icon image makes it clear.